src/components/forms/VerifyIdentityInput.tsx

```
@@ -0,0 +1,103 @@
+    }
+  };
+
+  const handleClick = () => {
+    inputRef.current?.click();
+  };
+
+  return (
+    <div className="space-y-2">
+      <label className="text-bodySmall text-neutral-80 font-medium">
+        {label}
+      </label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)